src/modules/cmdpal/ext/Microsoft.CmdPal.Ext.TimeDate/FallbackTimeDateItem.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using Microsoft.CmdPal.Ext.TimeDate.Helpers;
+using Microsoft.CommandPalette.Extensions;
+using Microsoft.CommandPalette.Extensions.Toolkit;
+
+namespace Microsoft.CmdPal.Ext.TimeDate;
+
+internal sealed partial class FallbackTimeDateItem : FallbackCommandItem
+{
+    private readonly HashSet<string> _validOptions;
+    private SettingsManager _settingsManager;
+
+    public FallbackTimeDateItem(SettingsManager settings)
+         : base(new NoOpCommand(), Resources.Microsoft_plugin_timedate_fallback_display_title)
+    {
+        Title = string.Empty;
+        Subtitle = string.Empty;
+        _settingsManager = settings;
+        _validOptions = new(StringComparer.OrdinalIgnoreCase)
+        {
+            "week", "year", "now", "time",
+        };
+    }
+
+    public override void UpdateQuery(string query)
+    {
+        if (string.IsNullOrWhiteSpace(query) || !_validOptions.Contains(query))
+        {
+            Title = string.Empty;
+            Subtitle = string.Empty;
+            Command = new NoOpCommand();
+            return;
+        }
+
+        var availableResults = AvailableResultsList.GetList(true, _settingsManager);
```

Comment:
Don't have the method params in mind. But if you set keywordsearch param to false you only get the results time, date and now. (The week number is not included. But we can move that result.) That would be the same behavior like PT Run has.